packages/tests/src/node/node.ts

```diff
@@ -210,14 +210,13 @@ export class NimGoNode {
   }
 
   async ensureSubscriptions(
-    pubsubTopics: [string] = [DefaultPubSubTopic]
+    pubsubTopics: string[] = [DefaultPubSubTopic]
   ): Promise<boolean> {
     this.checkProcess();
 
-    return this.rpcCall<boolean>(
-      "post_waku_v2_relay_v1_subscriptions",
+    return this.rpcCall<boolean>("post_waku_v2_relay_v1_subscriptions", [
       pubsubTopics
```

Comment:
this does not seem correct because you are now sending an array of array `string[][]`